streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -290,11 +291,40 @@ private void shutdown() {
             log.error("stream-thread [{}] Failed to close restore consumer: ", this.getName(), e);
         }
 
+        // remove all tasks
         removeStreamTasks();
+        removeStandbyTasks();
```

Comment:
I think we still need to do: 
```
 // un-assign the change log partitions
 restoreConsumer.assign(Collections.<TopicPartition>emptyList());
```
in this method. The assignment happens in `StreamThread.addStandbyTasks` and i don't see them being removed anywhere else